meerk40t/gui/simulation.py

```diff
@@ -590,9 +796,13 @@ def slide_out(self, event):
 
     def on_redo_it(self, event):
         with wx.BusyInfo(_("Preparing simulation...")):
+            if self.checkbox_optimize.GetValue():
+                opt = " optimize"
+            else:
+                opt = ""
             self.context(
-                "plan{plan} clear\nplan{plan} copy preprocess validate blob preopt optimize\n".format(
-                    plan=self.plan_name
+                "plan{plan} clear\nplan{plan} copy preprocess validate blob preopt{optimize}\n".format(
```

Comment:
This can't be translated and could just be an fstring if we're adding stuff to it like this.

Reply:
Done